### PR TITLE
Added: Modal Component [P1]

### PR DESCRIPTION
**Pull request by samflab**
please review this component. thank you

deployed at: https://deploy-preview-11--azure-ui.netlify.app/modal/modal

### CODE REVIEW

**Review by NtshVrm — APPROVED** [review]

**Review by shraddha15-creator — APPROVED** [review]

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (rutvikpumak) [inline .vscode/settings.json:1]: Remove this file

File: modal/modal.css [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  border-radius: 5px;
+  box-shadow: #183040 1.95px 1.95px 2.6px;
+  margin: auto;
+  visibility: hidden;
+  width: 21vw;
+}
+
+.header-title {
+  font-size: 1.2rem;
+  font-weight: 500;
```
Comment (rutvikpumak) [inline modal/modal.css:24]: Again use local variable for font-weight

File: modal/modal.css [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+  font-size: 1rem;
+  padding: 0rem 1rem;
+}
+
+.button-container{
+  display: flex;
+  flex-direction: row;
+  justify-content: space-evenly;
+  padding: 1rem 1rem;
+}
```
Comment (rutvikpumak) [inline modal/modal.css:38]: Shift your button to the end of the container and also reduce the size of the button. It's looking big according to the container. By default, it's flex-direction row so remove that line. Use justify-content:flex-end here and give some gap

Comment (samflab) [reply modal/modal.css:38]: Thank you! done. 😃

**Pull request merged**